src/KeyboardInputModal.scss

```diff
@@ -0,0 +1,164 @@
+@import '_variables';
+
+.modal {
+    border-radius: 0.25rem;
+}
+
+.modal-body {
+    border-radius: 0.25rem;
+    padding: 0;
+}
+
+.KeyboardInputModal {
+    max-width: 75%;
+}
+
+.KeyboardInputModal__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    row-gap: .5rem;
+}
+
+.KeyboardInputModal__content__title {
+    background-color: #F1F2F4;
+    border-radius: 0.25rem 0.25rem 0 0;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    column-gap: 0.5rem;
+    width: 100%;
+    padding: 0.5rem;
+
+    svg {
+        width: 2rem;
+    }
+}
+
+.KeyboardInputModal__content__toggleBar {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    column-gap: 2rem;
+    margin-left: 3rem;
+}
+
+.KeyboardInputModal__content__toggleBar__label {
+    font-weight: bold;
+}
+
+.KeyboardInputModal__content__toggleBar__toggle {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    column-gap: 0.5rem;
+}
+
+.KeyboardInputModal__content__toggle.ToggleSwitch {
+    .ToggleSwitch__switch-inner-circle {
+        height: 1.9rem;
+        width: 1.9rem;
+    }
+}
+
+.KeyboardInputModal__content__toggle.ToggleSwitch--checked {
+    background-color: #4C9990;
+}
+
+.KeyboardInputModal__content__schemeDropdown {
+    margin-bottom: 0.5rem;
+    margin-left: 3rem;
+    padding: 0.25rem;
+    width: 20rem;
+}
+
+.KeyboardInputModal__content__list {
+    background-color: #F1F2F4;
+    padding: 1rem;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 1.5rem;
+    row-gap: 1.5rem;
+}
+
+@media only screen and (max-width: 1024px) {
+    .KeyboardInputModal__content__list {
+        display: flex;
+        flex-direction: column;
+        column-gap: 0;
+    }
+}
+
+.KeyboardInputModal__content__list--disabled {
+    display: none;
+}
+
+.KeyboardInputModal__binding {
+    display: grid;
+    grid-template-columns: 6rem auto;
```

Comment:
6rem is not enough width to render 3 key combinations, which result them to look squished. Each icon is 2.5rem width, so we would need to change this value to 5rem on 2 key combinations layout and 7.5rem on 3 key combinations. Setting this to min-contnet would be much easier, but we have ? icon, which doesn't have same number of icons in a row as other commands.

Reply:
I ended up making the column 7.5rem for now, and it looks better.  There is a small concern if we start supporting 4-key combos (`shift+ctrl+alt+X`), but we can discuss that when we have to.